### PR TITLE
#15236: fix redundent scrollbars on UI

### LINKED ISSUE #15236: redundent scrollbars on UI for description field
redundent scrollbars are showing in right panel for no description field and horizontal scrollbar for right panel

for reffernce you can check below screenshot

please have a look into this

Screenshot 2024-02-19 at 1 42 02 PM

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes:

Fixes #15236

Issue : 
image

image

Resolved :
image

image
image

#
### Type of change:

- [x] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/Entity/Task/TaskTab/task-tab.less [hunk 1]
```diff
@@ -19,15 +19,17 @@
...
+    align-items: center;
+    flex-wrap: wrap;
 
-  &:hover {
-    .ant-typography {
-      color: @primary-color;
+    &:hover {
+      .ant-typography {
+        color: @primary-color;
+      }
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/Entity/Task/TaskTab/task-tab.less:32]: this style seems to be repeating, can you check?

**Review by ShaileshParmar11 — COMMENTED** [review]
one comment

File: openmetadata-ui/src/main/resources/ui/src/components/Entity/Task/TaskTab/task-tab.less (hunk 1 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/Entity/Task/TaskTab/task-tab.less:32]: @ShaileshParmar11 so, we have this style but both of them are not independent, they are used with the hierarchy of their parent.

Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/Entity/Task/TaskTab/task-tab.less]: @ShaileshParmar11  make the CSS resuable from a single location!

**Review by ShaileshParmar11 — APPROVED** [review]

**Review by karanh37 — APPROVED** [review]

**Pull request merged**